modules/mono/editor_templates/CharacterBody2D/basic_movement.cs

```diff
@@ -0,0 +1,41 @@
+// meta-description: Classic movement for gravity games (platformer, ...)
+// meta-default: true
+
+using _BINDINGS_NAMESPACE_;
+using System;
+
+public partial class _CLASS_ : _BASE_
+{
+    public const int Speed = 1000;
+    public const int JumpForce = -1000;
+
+    // Get the gravity from the project settings to be synced with RigidDynamicBody nodes.
+    public int gravity = (int)ProjectSettings.GetSetting("physics/2d/default_gravity");
+
+    public override void _PhysicsProcess(float delta)
+    {
+        Vector2 motionVelocity = MotionVelocity;
+
+        // Add the gravity.
+        if (!IsOnFloor())
+            motionVelocity.y += gravity * delta;
+
+        // Handle Jump.
+        if (Input.IsActionJustPressed("move_up") && IsOnFloor())
```

Comment:
The action `move_up` does not exist, in GDScript you are using `ui_accept`.

Same problem in the 3D template.